nilearn/reporting/get_clusters_table.py

```diff
@@ -318,7 +318,7 @@ def get_clusters_table(
     stat_map = safe_get_data(
         stat_img,
         ensure_finite=True,
-        copy_data=(cluster_threshold is not None),
+        copy_data=(cluster_threshold != 0),
```

Comment:
I've changed the condition at this line as at line 298:

```
cluster_threshold = 0 if cluster_threshold is None else cluster_threshold
```

So `cluster_threshold` can never be None at this line.